src/components/ComponentPages/Login/index.tsx

```diff
@@ -41,6 +41,16 @@ const Login = ({ isModal }) => {
     }
   };
 
+  const onOTPClick = async (e) => {
+    e.preventDefault();
+    const emailPhone = form.getFieldValue("username");
+    if (emailPhone?.trim()) {
+      console.log("emailPhone", emailPhone);
```

Comment:
Do we need this?

Reply:
after integration of Login with OTP API we will remove this